pyapp/{{ cookiecutter.slug }}/{{ cookiecutter.package_name }}/__main__.py

```diff
@@ -0,0 +1,5 @@
+import sys
+
+from {{ cookiecutter.package_name }}.app import run
+
+sys.exit(run())
```

Comment:
This just makes the package runnable with `python3 -m {{ cookiecutter.package_name }}`, which we use in `supervisord{-dev}.conf`